site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
+| Deep extreme <br>cut (DEXTR)                              | This is an optimized version of the original model, <br>introduced at the end of 2017. It uses the <br>information about extreme points of an object <br>to get its mask. The mask is then converted to a polygon. <br>For now this is the fastest interactor on the CPU.                                                         | ![](/images/dextr_example.gif) |
+| Feature backpropagating <br>refinement <br>scheme (f-BRS) | The model allows to get a mask for an <br>object using positive points (should be <br>left-clicked on the foreground), <br>and negative points (should be right-clicked <br>on the background, if necessary). <br>It is recommended to run the model on GPU, <br>if possible.                                                     | ![](/images/fbrs_example.gif)  |
+| High Resolution <br>Net (HRNet)                           | The model allows to get a mask for <br>an object using positive points (should <br>be left-clicked on the foreground), <br> and negative points(should be <br>right-clicked on the background, <br> if necessary). <br>It is recommended to run the model on GPU, <br>if possible.                                                | ![](/images/hrnet_example.gif) |
+| Inside-Outside-Guidance<br>(IOG)                          | The model uses a bounding box and <br>inside/outside points to create a mask. <br>First of all, you need to create a bounding<br> box, wrapping the object. <br>Then you need to use positive <br>and negative points to say the <br>model where is <br>a foreground, and where is a background.<br>Negative points are optional. | ![](/images/iog_example.gif)   |
 
-![](/images/iog_example.gif)
+<!--lint enable maximum-line-length-->
 
-## Detectors
+### Detectors
 
-Detectors are used to automatically annotate one frame. Supported DL models are suitable only for certain labels.
+Use detectors to automatically annotate one frame.
 
-- Before you start, click the `magic wand` on the controls sidebar and select the `Detectors` tab.
-  You need to match the labels of the DL model (left column) with the labels in your task (right column).
-  Then click `Annotate`.
+Supported DL models are suitable only for certain labels.
 
-  ![](/images/image187.jpg)
+To annotate with detectors, do the following:
 
-- Some of models supports attributes annotation (like facial emotions, for example: ``serverless/openvino/omz/intel/face-detection-0205``).
-  In this case you can also match attributes of the DL model with the attributes of a CVAT label.
+1. Click **Magic wand** ![Magic wand](/images/image189.jpg), and go to the **Detectors** tab.
+2. From the **Model** drop-down, select model (see [Detectors Models](#detectors-models)).
+3. From the left drop-down select the DL model label and match it to the right drop-down - the label of your task.
```

Comment:
I will recommend to add an example. For example, a DL model can have 'car' label, but our task or project can have `vehicle` label. Thus, we need to give a hint to our DL model (detector) what we want to annotate cars on images.